plan/planbuilder.go

```diff
@@ -494,6 +493,50 @@ func (b *planBuilder) buildAdmin(as *ast.AdminStmt) Plan {
 	return ret
 }
 
+func (b *planBuilder) buildAdminCheckTable(as *ast.AdminStmt) Plan {
+	p := &CheckTable{Tables: as.Tables}
+	p.GenExprs = make(map[string]expression.Expression, 0)
+
+	mockTablePlan := LogicalTableDual{}.init(b.ctx)
```

Comment:
Why use `LogicalTableDual`?

Reply:
I mock from `buildInsert()`
`LogicalTableDual` is use to `b.rewrite(column.GeneratedExpr, mockTablePlan, nil, true)`

Reply:
So what's the problem with using CheckTable? What about change CheckTable to

```
type CheckTable struct {
	logicalSchemaProducer

	Tables []*ast.TableName

	GenExprs map[string]expression.Expression
}
```

Reply:
So what's the problem with using CheckTable?

Reply:
`CheckTable` don't implement `LogicalPlan` , `expr, _, err := b.rewrite` need a LogicalPlan